CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ option(CHFL_CODE_COVERAGE "Enable code coverage" OFF)
 option(CHFL_USE_WARNINGS "Compile the code with warnings (default in debug mode)" OFF)
 option(CHFL_DEBUG_GLIBCXX "Use the debug version of GNU libstdcxx" OFF)
 option(CHFL_SYSTEM_NETCDF "Use the system NetCDF instead of the internal one" OFF)
+option(CHFL_SYSTEM_ZLIB "Use the system zlib instead of the internal one" OFF)
```

Comment:
Do we need CI for this? Probably not, but worth a discussion.

Reply:
I don't know. We have one builder on Travis testing with the system NetCDF, we could use it with the system zlib and lzma too.